requirements/constraints.txt

```diff
@@ -82,3 +86,6 @@ aiofile>=999999999
 # directly use http calls.
 # SEE https://github.com/Pennsieve/pennsieve-python/issues/17
 pennsieve>=999999999
+
+# User alternative e.g. parametrized fixture or request.getfixturevalue(.)
+pytest-lazy-fixture>=999999999
```

Comment:
I guess you needed it here to upgrade and remove it.
Is it still required?

Reply:
@GitHK this is how you avoid this library to be ever installed here. 
![image](https://github.com/ITISFoundation/osparc-simcore/assets/32402063/6f2eb96e-42bc-45bb-b65c-d1bc0f5258fa)